docs/produce-custom-themes-using-asciidoctor-stylesheet-factory.adoc

```diff
@@ -151,6 +151,10 @@ include::{includedir}/apply-theme.adoc[]
 
 Happy theming!
 
+== Additional themes for Asciidoctor
+
+The https://github.com/darshandsoni/asciidoctor-skins[asciidoctor-skins] repository has some additional custom stylesheets that can be used to theme asciidoctor documentation. These are specifically created for users who want some additional choice in styling their documents but do not want to dive into customising css. The custom stylesheets in asciidoctor-skins import the default stylesheet first, and then customize specific elements.
```

Comment:
To harmonize spelling in the paragraph: `customising css` => `customizing CSS`

(In addition to picking US English spelling, I changed the lower-case to the upper-case of "CSS", as this is the most-common spelling.)